x/incentives/keeper/distribute.go

```diff
@@ -258,6 +263,32 @@ func (k Keeper) distributeSyntheticInternal(
 	return k.distributeInternal(ctx, gauge, sortedAndTrimmedQualifiedLocks, distrInfo)
 }
 
+// distributeCLIncentiveInternal runs the distribution logic for CL pools only. It creates new incentive record with osmo incentives
+// and distributes all the tokens to the dedicated pool
+func (k Keeper) distributeCLIncentiveInternal(ctx sdk.Context, incentiveRecord cltypes.IncentiveRecord, gauge types.Gauge) (sdk.Coins, error) {
```

Comment:
can we also add unit tests for this method where under different conditions we check that
1. incentive was created
2. gauge has been correctly updated

Reply:
these checks has already been added as a part of TestDistributeToConcentratedLiquidityPools